invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019-2022 CERN.
+# Copyright (C) 2019-2022 Northwestern University.
+# Copyright (C)      2022 TU Wien.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""Request views module."""
+
+from flask import render_template
+from flask_login import login_required
+from jinja2 import TemplateNotFound
+from sqlalchemy.orm.exc import NoResultFound
+from flask import g
+from invenio_requests.views.decorators import pass_request
+from invenio_rdm_records.resources.serializers import UIJSONSerializer
+from invenio_rdm_records.proxies import current_rdm_records_service
+
+
+@login_required
+@pass_request
+def requests_detail(request=None, pid_value=None):
+    """Community detail page."""
+    request_dict = request.to_dict()
+
+    # temporarily, until serializers and avatars implemented
+    try:
+        user_email = request_dict["created_by"]["user"]
+        request_dict["created_by"].update({
+            "avatar": "/static/images/placeholder.png",
+            "full_name": user_email
+        })
+    except KeyError:
+        request_dict["created_by"].update({
+            "avatar": "/static/images/placeholder.png",
+            "full_name": request_dict["created_by"]["community"]
+        })
+
+    topic_record = None
+    is_draft = None
+    permissions = []
+
+    if 'record' in request_dict["topic"]:
```

Comment:
Can you change this to a named constant? what does it mean "having a record in a topic"?